src/mirage_kv_unix.ml

```diff
@@ -128,15 +122,15 @@ let get_aux { base } ?offset ?length key =
       | Internal_error e -> Lwt.return (Error e)
       | Unix.Unix_error (ENOENT, _, _) -> Lwt.return (err_not_found key)
       | Unix.Unix_error (e, _, _) -> Lwt.return (err_unix key e)
-      | e -> Lwt.fail e)
+      | e -> Lwt.reraise e)
```

Comment:
`Lwt.reraise` is rather recent and introduced in lwt.5.7.0 so we should reflect that in mirage-kv-unix.opam

Reply:
Good point, I've bumped the version and force-pushed